juju/juju
# Remove inbuilt dashboard since we are moving to use a separate charm

We are moving to deploy the juju dashboard using a charm instead of it being served by the controller.
This means that the workflow and CLI around locating and pulling a dashboard tarball from simplestreams and uploading it to the controller and extracting it and providing http endpoints on the controller API etc are all no longer needed.

Also tweak the controller-url config for the controller charm to include the port.

## QA steps

juju bootstrap --no-default-model
(see that no dashboard messages are logged)

```
juju config controller
...
charm: juju-controller
settings:
  controller-url:
    default: ""
    description: The URL used to access the controller API.
    source: user
    type: string
    value: wss://10.24.226.7:17070/api
...
```

## Bug reference

https://bugs.launchpad.net/juju/+bug/1945363
